spec/controllers/trainer_controller_spec.rb

```diff
@@ -0,0 +1,228 @@
+require 'rails_helper'
+require 'support/helpers/trainer_helper.rb'
+include TrainerHelper
+
+describe TrainerController do
+  describe 'review_card' do
+    describe 'correct translation' do
+      before do
+        @user = create(:user)
+        @block = create(:block, user: @user)
+        @controller.send(:auto_login, @user)
+      end
+
+      it 'set review_date step=1' do
+        card = create_and_check_review_card(@user, @block, 1, 'house')
+        expect(card.review_date.strftime('%Y-%m-%d %H:%M')).
+            to eq((Time.zone.now + 12.hours).strftime('%Y-%m-%d %H:%M'))
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.